inttest/cnichange/cnichange_test.go

```diff
@@ -44,11 +44,12 @@ func (s *CNIChangeSuite) TestK0sGetsUpButRejectsToChangeCNI() {
 	s.Require().NoError(err)
 
 	// Wait till we see kube-router DS created
-	watch.DaemonSets(kc.AppsV1().DaemonSets("kube-system")).
+	err = watch.DaemonSets(kc.AppsV1().DaemonSets("kube-system")).
 		WithObjectName("kube-router").
 		Until(s.Context(), func(ds *appsv1.DaemonSet) (bool, error) {
 			return true, nil
 		})
+	s.NoError(err)
```

Comment:
I wonder if we wanna use Require() here and in the other places.

Reply:
I wonder if we can somehow change the embedding to use Require by default, not the Assert

Reply:
I used the assert one in the errors which were ignored because we weren't failing on error, but actually it makes sense to fail early so I'm changing all of these.